meshtastic-mqtt-connect.py

```diff
@@ -632,14 +635,19 @@ def send_position(destination_id):
 
 
 def generate_mesh_packet(destination_id, encoded_message):
+    global global_message_id
     mesh_packet = mesh_pb2.MeshPacket()
 
+    # Use the global message ID and increment it for the next call
+    mesh_packet.id = global_message_id
+    global_message_id += 1
+    
     setattr(mesh_packet, "from", node_number)
-    mesh_packet.id = random.getrandbits(32)
     mesh_packet.to = destination_id
     mesh_packet.want_ack = False
     mesh_packet.channel = generate_hash(channel, key)
     mesh_packet.hop_limit = 3
+    mesh_packet.rx_time = int(time.time())
```

Comment:
looks good, just have to loose this I think, unless you think I'm wrong on the other PR.